File: CS/App_Code/CustomLayoutDesigner.cs
using System.ComponentModel;
using DevExpress.Web.ASPxFormLayout;
using DevExpress.Web.ASPxFormLayout.Design;

namespace CustomLayout {
    [ToolboxItem(false)]
    public class CustomLayoutDesigner : ASPxFormLayoutDesigner {
        public override void Initialize(IComponent component) {
            base.Initialize(component);
            RegisterTagPrefix(typeof(ASPxFormLayout));
        }
    }
}

File: VB/App_Code/CustomLayoutDesigner.vb
Imports Microsoft.VisualBasic
Imports System.ComponentModel
Imports DevExpress.Web.ASPxFormLayout
Imports DevExpress.Web.ASPxFormLayout.Design

Namespace CustomLayout
	<ToolboxItem(False)> _
	Public Class CustomLayoutDesigner
		Inherits ASPxFormLayoutDesigner
		Public Overrides Sub Initialize(ByVal component As IComponent)
			MyBase.Initialize(component)
			RegisterTagPrefix(GetType(ASPxFormLayout))
		End Sub
	End Class
End Namespace